examples/spark/src/main/scala/org/apache/carbondata/examples/CarbonShowPartitionInfo.scala

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.carbondata.examples
+
+import scala.collection.mutable.LinkedHashMap
+
+import org.apache.carbondata.core.constants.CarbonCommonConstants
+import org.apache.carbondata.core.util.CarbonProperties
+import org.apache.carbondata.examples.util.ExampleUtils
+
+object CarbonShowPartitionInfo {
+  def main(args: Array[String]) {
+
+    CarbonShowPartitionInfo.extracted("t3", args)
+  }
+  def extracted(tableName: String, args: Array[String]): Unit = {
+    val cc = ExampleUtils.createCarbonContext("CarbonShowPartitionInfo")
+    val testData = ExampleUtils.currentPath + "/src/main/resources/data.csv"
+
+    // range partition
+    cc.sql("DROP TABLE IF EXISTS t1")
+
+    cc.sql("""
+                | CREATE TABLE IF NOT EXISTS t1
+                | (
+                | vin String,
+                | phonenumber Int,
+                | country String,
+                | area String
+                | )
```

Comment:
I think key words in sql command and data type should be upper case. 
for example, "String" -> "STRING"
anyway, it's not a big problem.